src/levenberg_marquardt.jl

```diff
@@ -0,0 +1,170 @@
+immutable LevenbergMarquardt <: Optim.Optimizer end
```

Comment:
That is certainly a good point. Actually, there is there any reason to have any Optim dependecy at all? I dont Think there is, so I should probably clean it up.